lib/mida/vocabulary.rb

```diff
@@ -41,8 +41,8 @@ def self.inherited(subclass)
     # This won't work properly until all the included_vocabularies
     # have finished being defined in case of circular dependancies
     def self.properties
-      return @properties if @properties
-      @properties = @this_properties || {}
+      return @properties if defined?(@properties)
```

Comment:
What about when `@properties` is `nil`. This used to be false, but now would be true.